application/Modules/Invoices/Models/Invoice.php

```
@@ -435,16 +486,24 @@ public function get_date_due($invoice_date_created)
+     *
+     * Legacy migration info:
+     * @legacy-file application/modules/invoices/models/Mdl_invoice.php
+     * @legacy-function get_invoice_number()
      */
     public function get_invoice_number($invoice_group_id)
     {
-        $this->load->model('invoice_groups/mdl_invoice_groups');
+        $this->load->model('invoice_groups/invoice_group');
 
         return $this->mdl_invoice_groups->generate_invoice_number($invoice_group_id);
     }
```

PHPMD (2.15.0)

The method get_invoice_number is not named in camelCase. (undefined)

(CamelCaseMethodName)